client/app/components/packages/pas-form/show.hbs

```diff
@@ -0,0 +1,383 @@
+        </strong>
+        <br>
+        {{if (eq @package.pasForm.dcpProjectareaischancefloodplain "717170000") "Yes"}}
+        {{if (eq @package.pasForm.dcpProjectareaischancefloodplain "717170001") "No"}}
+        {{if (eq @package.pasForm.dcpProjectareaischancefloodplain "717170002") "Unsure at this time"}}
+      </p>
+
+      <p>
+        <strong>
+          Is a <a href="#">legal instrument</a> associated with a previous CPC or CPC Chair action recorded against the project site?
+          {{!-- TODO: add link --}}
+        </strong>
+        <br>
+        {{if (eq @package.pasForm.dcpRestrictivedeclaration true) "Yes"}}
+        {{if (eq @package.pasForm.dcpRestrictivedeclaration false) "No"}}
+      </p>
+
+      <p>
+        <strong>
+          City Register File Number
+        </strong>
+        <br>
+        {{@package.pasForm.dcpCityregisterfilenumber}}
+      </p>
+
+      <p>
+        <strong>
+          Will a new legal instrument or modification of a current legal instrument be necessary to approve/implement these actions?
+        </strong>
+        <br>
+        {{if (eq @package.pasForm.dcpRestrictivedeclarationrequired "717170000") "Yes"}}
+        {{if (eq @package.pasForm.dcpRestrictivedeclarationrequired "717170001") "No"}}
+        {{if (eq @package.pasForm.dcpRestrictivedeclarationrequired "717170002") "Unsure at this time"}}
+      </p>
+    </section>
+
+    <section class="form-section" id="#proposed-development-site">
+      <h3>Proposed Development Site</h3>
+
+      <p>
+        <strong>
+          In what year do you expect the development to complete?
+        </strong>
+        <br>
+        {{@package.pasForm.dcpEstimatedcompletiondate}}
+      </p>
+
+      <p>
+        <strong>
+          What Type of Development is Proposed?
+        </strong>
+        <br>
+        {{if @package.pasForm.dcpProposeddevelopmentsitenewconstruction "Newly constructed buildings"}}
+        {{if @package.pasForm.dcpProposeddevelopmentsitedemolition "Demolition"}}
+        {{if @package.pasForm.dcpProposeddevelopmentsiteinfoalteration "Alteration"}}
+        {{if @package.pasForm.dcpProposeddevelopmentsiteinfoaddition "Addition"}}
+        {{if @package.pasForm.dcpProposeddevelopmentsitechnageofuse "Change of use"}}
+        {{if @package.pasForm.dcpProposeddevelopmentsiteenlargement "Enlargement"}}
+        {{if @package.pasForm.dcpProposeddevelopmentsiteinfoother "Other: "}}
+        {{if @package.pasForm.dcpProposeddevelopmentsiteinfoother @package.pasForm.dcpProposeddevelopmentsiteotherexplanation}}
+      </p>
+
+      <p>
+        <strong>
+          Is the Development Site in an <a href="#">Inclusionary Housing Designated Area/Mandatory Inclusionary Housing Area</a>?
+          {{!-- TODO: add link --}}
+        </strong>
+        <br>
+        {{if (eq @package.pasForm.dcpIsinclusionaryhousingdesignatedarea true) "Yes"}}
+        {{if (eq @package.pasForm.dcpIsinclusionaryhousingdesignatedarea false) "No"}}
```

Comment:
I could see a helper for stuff like this maybe `{{humanize @package.pasForm.dcpIsinclusionaryhousingdesignatedarea}}`

Idk.